### PR TITLE
Add new tests for get APIs

### PR DESCRIPTION
**Pull request by pgiffy**

### CODE REVIEW

File: test/admin_tests.js [hunk 1]
```diff
@@ -29,7 +28,7 @@ after(function() {
     process.exit(0);
 });
 
-describe('/GET admins', () => {
+describe('Admin API suitr /GET,/REGISTER,/LOGIN admins', () => {
```
Comment (sambehrens) [inline test/admin_tests.js]: you spelled suite wrong

File: test/admin_tests.js [hunk 2]
```diff
@@ -41,9 +40,7 @@ describe('/GET admins', () => {
                 done();
             });
     });
-});
 
-describe('/REGISTER admins', () => {
```
Comment (sambehrens) [inline test/admin_tests.js:46]: admins should be singular here because you only register one admin

File: test/admin_tests.js [hunk 3]
```diff
@@ -62,9 +59,7 @@ describe('/REGISTER admins', () => {
                 done();
             });
     });
-});
 
-describe('/LOGIN admins', () => {
```
Comment (sambehrens) [inline test/admin_tests.js:67]: same comment here as register admins

File: test/admin_tests.js (hunk 1 above)
Comment (pgiffy) [reply test/admin_tests.js]: I changed it

**Review by sambehrens — APPROVED** [review]

File: test/admin_tests.js (hunk 3 above)
Comment (pgiffy) [reply test/admin_tests.js:67]: Its deleted

**Pull request merged**